MagnetoSoftwareTesting/src/test/java/com/utility/ExcelUtil.java

```diff
@@ -0,0 +1,32 @@
+package com.utility;
+
+import java.util.ArrayList;
+
+import com.excel.Utility.Xls_Reader;
+
+public class ExcelUtil {
+	static Xls_Reader reader;
+
+	public static ArrayList<Object[]> getDataFromExcel() {
+		ArrayList<Object[]> myData = new ArrayList<Object[]>();
+		try {
+			reader = new Xls_Reader(
+					"src\\test\\java\\ExcelData\\data .xlsx");
+		} catch (Exception e) {
+			e.printStackTrace();
```

Comment:
Add proper loggers to print error message if there are any.

Reply:
added logger.